public_html/twoDimArr.js

```diff
@@ -0,0 +1,31 @@
+
+function randCrate(){
+    let color = ["R", "B", "Y", "G"];
+    let crateCol = color[parseInt(Math.floor(Math.random()*4))];
+    let crateWt = Math.ceil(Math.random()*9);
+    if(crateWt === 0) crateWt ++;
```

Comment:
Normally, it's better to avoid special cases and make a general implementation. `let crateWt = Math.floor(Math.random() * 9 ) + 1;` will eliminate this line.